api_server/charity/tests/test_charity.py

```diff
@@ -0,0 +1,174 @@
+    @pytest.mark.asyncio
+    async def test_try_to_create_charity_without_auth_should_be_denied(self,
+                                                                       app: FastAPI,
+                                                                       client: AsyncClient,
+                                                                       test_user: User
+                                                                       ):
+        """
+        Test POST '/charities' endpoint with NOT authenticated user.
+
+               Args:
+                   app: pytest fixture, an instance of FastAPI.
+                   client: pytest fixture, an instance of AsyncClient for http requests.
+                   test_user: pytest fixture, add user to database.
+
+               Returns:
+
+        """
+        url = app.url_path_for('create_charity')
+        response = await client.post(url, json=response_create_organisation_endpoint(test_user.id))
+        assert response.status_code == HTTP_401_UNAUTHORIZED
+        assert response.json() == UNAUTHORIZED
+
+    @pytest.mark.asyncio
+    async def test_create_charity_should_be_ok(
+            self, app: FastAPI, client: AsyncClient, db_session: AsyncSession, authenticated_test_user: User
+    ):
+        """
+        Test POST '/charities' endpoint with authenticated user.
+
+        Args:
+            app: pytest fixture, an instance of FastAPI.
+            client: pytest fixture, an instance of AsyncClient for http requests.
+            db_session: pytest fixture, sqlalchemy AsyncSession.
+            authenticated_test_user: pytest fixture, add user to database and add auth cookies to client fixture.
+
+        Returns:
+
+        """
+
+        url = app.url_path_for('create_charity')
+        response = await client.post(url, json=response_create_organisation_endpoint(authenticated_test_user.id))
+
+        response_data = response.json()
+        organisation_id = response_data["data"][0]["id"]
+        expected_result = get_successful_organisation_creating(organisation_id)
+        organisation = (await db_session.execute(select(CharityOrganisation)
+                                                 .where(CharityOrganisation.id == organisation_id))).scalar_one()
+        assert response.status_code == HTTP_201_CREATED
+        assert response_data == expected_result
+        assert organisation.users[0].username == "test_john"
+
+    @pytest.mark.asyncio
+    async def test_edit_charity_not_authenticated_should_be_denied(self,
+                                                                   app: FastAPI,
+                                                                   client: AsyncClient,
+                                                                   charity: CharityOrganisation
+                                                                   ):
+        """
+        Test PUT '/charities/{org_id}' endpoint with NOT authenticated user.
```

Comment:
Few docstrings here and below looks incomplete.

Reply:
I've written more comprehensive description on this docstring